index.js

```diff
@@ -22,6 +31,9 @@ function checkAltText(ast, file) {
       if (node.alt && altText(node.alt)) {
         file.message(altText(node.alt), node);
       }
+      if (imageIsLink) {
+        file.message("Alt text should describe the link, not the image.", node);
```

Comment:
@jsnmrs do you think there's a better way to word this?

Reply:
"Images inside of a link tag require alt text that describes the purpose of the link"